torch/_inductor/codecache.py

```diff
@@ -1728,10 +1730,23 @@ def _compile_consts_darwin(consts: bytes) -> str:
             )
 
             output_o = os.path.splitext(input_path)[0] + ".o"
+
+            all_cuda = all(
```

Comment:
I originally proposed to skip alignment for all_cuda case, but it seems unnecessarily complex. Ok to stick with it for this PR, but I will create a followup PR to skip all_cuda check, and verify if it looks fine for GPU.